quipucords/fingerprinter/__init__.py

```diff
@@ -12,15 +12,35 @@
 """Fingerprint engine ingests raw facts and produces system finger prints."""
 
 import logging
+import uuid
 from datetime import datetime
 import django.dispatch
 from api.fact.util import read_raw_facts
-from api.models import FactCollection
+from api.models import FactCollection, Source
 from api.serializers import FingerprintSerializer
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
-# pylint: disable=unused-argument
+NETWORK_IDENTIFICATION_KEYS = ['subscription_manager_id',
```

Comment:
Do we need a `VCENTER_IDENTIFICATION_KEYS` for dedup with `vm_uuid`? Are we doing this with another issue? If so, can you open the issue?

Reply:
Added and there is de-dup code